src/Mock.Chemistry/Library.qs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+///
```

Comment:
[nit] Since this is a `///` comment, it would be good for it to start with a `/// # Summary` line. Alternatively, could make them into ordinary comments instead.